lib/semian/mysql2.rb

```diff
@@ -40,6 +40,9 @@ module Mysql2
       /\A\s*ROLLBACK/i,
       /\A\s*COMMIT/i,
       /\A\s*RELEASE\s+SAVEPOINT/i,
+      /ROLLBACK[\s;]*\z/i,
```

Comment:
Wouldn't it be better to just ignore the comment?, e.g. `\A(?:\/\*.*?\*\/)?\s* ROLLBACK`.